src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateNewImage.cs

```diff
@@ -124,6 +124,20 @@ internal async Task<bool> ExecuteAsync(CancellationToken cancellationToken)
 
         SafeLog(Strings.ContainerBuilder_StartBuildingImage, Repository, string.Join(",", ImageTags), sourceImageReference);
 
+        // forcibly change the media type if required
+        if (ImageFormat is not null)
```

Comment:
nit: one if statement
if (ImageFormat != null && Enum.TryParse<KnownImageFormats>(ImageFormat, out var imageFormat))

Reply:
Ah, I think I know what I was intending to do here: if a user specifies some value that isn't a parseable image format, I need the separate if/else to know if I should raise an error. I need to add

```csharp
else {
  // raise error telling user that the value of `ImageFormat` is invalid
}
```

in order to make this block of code make sense.